tools/dgeni/src/processors/createAnchorTags.ts

```diff
@@ -0,0 +1,37 @@
+import { Document, Processor } from "dgeni";
+
+var INLINE_LINK_REGEX = /{@link.[^}]*}/g;
+const API_DOCS_PATH = "docs/api/";
+
+/**
+ * Process inline link tags (of the form {@link product/DaffProductFacade#product$ the product$ field}), and replace them with HTML anchors.
+ */
+export class CreateAnchorTagsProcessor implements Processor {
+	name = 'createAnchorTags';
+	$runAfter = ['docs-processed'];
+	$runBefore = ['rendering-docs'];
+
+	$process(docs: Document[]): Document[] {
+		docs.map(doc => {
+			if(!doc.members) return doc;
+			
+			doc.members.map(member => {
+				member.description = member.description.replace(INLINE_LINK_REGEX, buildAnchorTag)
+
+				return member;
+			})
+		})
+
+		return docs;
+	}
+}
+
+function buildAnchorTag(linkString): string {
+	let splitValues = linkString.split(' ');
+	//the title is always all of the values beyond the second part of the split values.
+	let title = splitValues.slice(2).join(' ');
```

Comment:
I think you could probably use regex capture groups to get these values a little more cleanly.

Reply:
changed